src/test/java/com/Project/accountManager/services/AccountServiceTest.java

```diff
@@ -0,0 +1,152 @@
+package com.Project.accountManager.services;
+
+import com.Project.accountManager.entities.Account;
+import com.Project.accountManager.entities.User;
+import com.Project.accountManager.repository.AccountRepository;
+import com.Project.accountManager.request.AccountCreateRequest;
+import com.Project.accountManager.request.AccountDepositRequest;
+import com.Project.accountManager.request.AccountWithdrawalRequest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
```

Comment:
never use wild card (.* )
here you can check why you should use wild card.
Maybe you can write one medium article about why shouldn't use wild card ?
What do u think  ?
Here some document
https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad
https://www.baeldung.com/java-wildcard-imports#:~:text=Wildcard%20imports%20help%20us%20avoid,code%20that%20shows%20the%20logic.

Also here about disabling wild card in intellij
https://www.google.com/search?q=intellij+disable+import+*&oq=intellij+disable&gs_lcrp=EgZjaHJvbWUqBwgBEAAYgAQyBggAEEUYOTIHCAEQABiABDIHCAIQABiABDIHCAMQABiABDIHCAQQABiABDIHCAUQABiABDIHCAYQABiABDIHCAcQABiABDIHCAgQABiABDIHCAkQABiABNIBCDUxMThqMGo3qAIAsAIA&sourceid=chrome&ie=UTF-8